evaporate.js

```diff
@@ -1847,10 +1800,46 @@
         return code.length ? {code: code, msg: msg} : {};
     }
 
+    var EvaporateHistoryCache = function () {
```

Comment:
If you're open to it, I can try setting up a webpack/browserify bundler so you can pack everything with one command. Also some `babel` with `es6` would be cool. Thoughts?